client/src/main/java/com/alibaba/nacos/client/naming/NacosNamingService.java

```diff
@@ -53,74 +55,74 @@
  */
 @SuppressWarnings("PMD.ServiceOrDaoClassShouldEndWithImplRule")
 public class NacosNamingService implements NamingService {
-    
-    private static final String DEFAULT_NAMING_LOG_FILE_PATH =  "naming.log";
-    
+
```

Comment:
Don't change indent. Please use nacos code style to reformat

Reply:
Ok.